incubator/command-dump/src/main/java/io/aklivity/zilla/runtime/command/dump/internal/airline/ZillaDumpCommand.java

```diff
@@ -146,6 +147,17 @@ private static byte[] hexStringToByteArray(String s)
     @Override
     public void run()
     {
+        if (directory == null)
+        {
+            try
+            {
+                directory = new URI(System.getProperty("user.dir"));
+            }
+            catch (URISyntaxException e)
+            {
+                System.out.println("Cannot parse working directory");
+            }
+        }
```

Comment:
Lets change the type of `directory` to `Path`, default it to `Paths.get(".zilla", "engine")` and remove this if block.